rules/docker-macsec.mk

```diff
@@ -42,6 +42,10 @@ $(DOCKER_MACSEC)_RUN_OPT += --privileged -t
 $(DOCKER_MACSEC)_RUN_OPT += -v /etc/sonic:/etc/sonic:ro
 $(DOCKER_MACSEC)_RUN_OPT += -v /host/warmboot:/var/warmboot
 
+$(DOCKER_MACSEC)_SERVICE_REQUIRES = updategraph
+$(DOCKER_MACSEC)_SERVICE_AFTER = swss syncd
```

Comment:
shouldn't gbsyncd too part of this?
gbsyncd is a vertical parallel to swss and syncd docker containers. Together the all three (swss, gbsyncd, syncd) complete interfaces related work-flow for SKU's having PHY (Gearbox)

Reply:
@jimmyzhai said gbsyncd is as a runtime service which cannot be bounded at make time. And gbsyncd has been bound to syncd, so we don't need to explicitly bind gbsyncd at here.